libc-test/build.rs

```diff
@@ -3994,6 +4014,13 @@ fn test_linux(target: &str) {
 
     cfg.skip_const(move |constant| {
         let name = constant.ident();
+
+        // L4Re requires a min stack size of 64k; that isn't defined in uClibc, but
+        // somewhere in the core libraries. uClibc wants 16k, but that's not enough.
+        if l4re && name == "PTHREAD_STACK_MIN" {
+            return true;
+        }
```

Comment:
Restructure this as

```rust
if l4re {
    match name {
        "PTHREAD_STACK_MIN" => true,
        _ => false,
    }
}
```

Which will make it easier to add more in the future if needed

Reply:
Unfortunately I can't since this is an early exit (involving a return statement). I could "return true" on the return case and do nothing on the catchall case, but I think it would be more clear code if we just left it as is?